public_profiler/v3.5-DD/ogl/skills.js

```diff
@@ -499,1857 +498,1866 @@ var skillsHelpURL = {
-  "razorfangs":                           baseWikiURL + "Razorfangs",
-  "reality maelstrom":                    baseWikiURL + "Reality_maelstrom",
-  "reaving dispel":                       baseWikiURL + "Reaving_dispel",
-  "rebuke, greater":                      baseWikiURL + "Greater_rebuke",
-  "rebuke":                               baseWikiURL + "Rebuke",
-  "rebuking breath":                      baseWikiURL + "Rebuking_breath",
-  "reciprocal gyre":                      baseWikiURL + "Reciprocal_gyre",
-  "recitation":                           baseWikiURL + "Recitation",
-  "red tide":                             baseWikiURL + "Red_tide",
-  "reduce person, greater":               baseWikiURL + "Greater_reduce_person",
-  "reflective disguise, mass":            baseWikiURL + "Mass_reflective_disguise",
-  "reflective disguise":                  baseWikiURL + "Reflective_disguise",
-  "refusal":                              baseWikiURL + "Refusal",
-  "regal procession":                     baseWikiURL + "Regal_procession",
-  "regroup":                              baseWikiURL + "Regroup",
-  "rejection":                            baseWikiURL + "Rejection",
-  "rejuvenating light":                   baseWikiURL + "Rejuvenating_light",
-  "rejuvenation cocoon":                  baseWikiURL + "Rejuvenation_cocoon",
-  "rejuvenative corpse":                  baseWikiURL + "Rejuvenative_corpse",
-  "remove scent":                         baseWikiURL + "Remove_scent",
-  "renewal pact":                         baseWikiURL + "Renewal_pact",
-  "renewed vigor":                        baseWikiURL + "Renewed_vigor",
-  "repair critical damage":               baseWikiURL + "Repair_critical_damage",
-  "repair light damage":                  baseWikiURL + "Repair_light_damage",
-  "repair minor damage":                  baseWikiURL + "Repair_minor_damage",
-  "repair moderate damage":               baseWikiURL + "Repair_moderate_damage",
-  "repair serious damage":                baseWikiURL + "Repair_serious_damage",
-  "replicate casting":                    baseWikiURL + "Replicate_casting",
-  "resist energy, mass":                  baseWikiURL + "Mass_resist_energy",
-  "resist negative energy":               baseWikiURL + "Resist_negative_energy",
-  "resist planar alignment":              baseWikiURL + "Resist_planar_alignment",
-  "resist positive energy":               baseWikiURL + "Resist_positive_energy",
-  "resist taint":                         baseWikiURL + "Resist_taint",
-  "resistance, greater":                  baseWikiURL + "Greater_resistance",
-  "resonating bolt":                      baseWikiURL + "Resonating_bolt",
-  "restful slumber":                      baseWikiURL + "Restful_slumber",
-  "restoration, mass":                    baseWikiURL + "Mass_restoration",
-  "resurgence, mass":                     baseWikiURL + "Mass_resurgence",
-  "resurgence":                     
```

Comment:
I would be happier with spaces on either side of : (and, above, of ?).

Reply:
That comment landed on the wrong line !  The : I was thinking of was the one that's part of a ?: ternary operator !